cmd/ipfs/daemon.go

```diff
@@ -135,7 +134,7 @@ Headers.
 		cmds.BoolOption(writableKwd, "Enable writing objects (with POST, PUT and DELETE)").Default(false),
 		cmds.StringOption(ipfsMountKwd, "Path to the mountpoint for IPFS (if using --mount). Defaults to config setting."),
 		cmds.StringOption(ipnsMountKwd, "Path to the mountpoint for IPNS (if using --mount). Defaults to config setting."),
-		cmds.BoolOption(unrestrictedApiAccessKwd, "Allow API access to unlisted hashes").Default(false),
+		cmds.BoolOption(unrestrictedApiAccessKwd, "This option has no effect since v0.4.3").Default(false),
```

Comment:
The option itself is still there, but not being used anymore, lines 365 ff. were the only usage. I left the option there for now so that you don't get `Error: Unrecognized option 'unrestricted-api'` if you were really using it. The new behaviour is as if `--unrestricted-api` was always true.